### ISSUE TITLE
AttributeError: type object 'tqdm_asyncio' has no attribute 'gather'

### ISSUE THREAD

**Issue by chenks** [DIGITALCRIMINAL/OnlyFans#1536]
hi

tried today to pull down videos and got the following error
any ideas what the problem is?

```
Scrape Attempt: 1/100
Type: Archived Posts
Processing Scraped Posts
Traceback (most recent call last):
  File "C:\of\start_ofd.py", line 60, in <module>
    loop.run_until_complete(main())
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.1776.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\of\start_ofd.py", line 44, in main
    api = await main_datascraper.start_datascraper(
  File "C:\of\datascraper\main_datascraper.py", line 93, in start_datascraper
    await main_helper.process_names(
  File "C:\of\helpers\main_helper.py", line 939, in process_names
    result = await module.start_datascraper(authed, username, site_name)
  File "C:\of\modules\onlyfans.py", line 160, in start_datascraper
    await prepare_scraper(authed, site_name, item)
  File "C:\of\modules\onlyfans.py", line 823, in prepare_scraper
    unrefined_set = await tqdm.gather(tasks, **settings)
AttributeError: type object 'tqdm_asyncio' has no attribute 'gather'
```

**Comment by LunarPenguin** [comment]
See: #1527 also see: #1521

**Comment by chenks** [comment]
Not sure those tell me anything?
I'm currently using the most recent release, and it was previously working fine, now all I get is the error I posted

**Comment by LunarPenguin** [comment]
make sure you have `tqdm` version `4.62.0` or greater installed. this can be done by the following command or something similar to it. `pip install -U tqdm>=4.62.0`

**Comment by chenks** [comment]
i've run the command you said above, and the same issue is happening.
i'm running this on a windows client.

```
C:\Users\chenk>python --version
Python 3.9.6

C:\Users\chenk>pip install -U tqdm>=4.62.0

C:\Users\chenk>
```

**Comment by LunarPenguin** [comment]
> C:\Users\chenk>pip install -U tqdm>=4.62.0

does this have any output?

you could also try running `pip3 install -U tqdm>=4.62.0` or `python -m pip install -U tqdm>=4.62.0`

You could also run `pip freeze` to see which version of packages you currently have installed.

**Comment by chenks** [comment]
`pip freeze` say

`tqdm==4.62.0`

**Comment by LunarPenguin** [comment]
Thanks for the info, i checked your library version and only see that `SQLAlchemy` is not the version specified in the `requirements.txt` file but i don't think that influences this problem.

I don't have any further steps for you to take to solve this problem but maybe @DIGITALCRIMINAL is able to help you further diagnose the problem.

**Comment by chenks** [comment]
yeah no idea why it's not working.
the auth.json looks all ok i think (i've x'ed out the auth_id).

it's able to pull down the avatar and header images, but nothing else.

```
{
  "auth": {
    "username": "default",
    "cookie": "csrf=81SPGI4N1d9144636b3a3033042e9f93e8b854a6; cookiesAccepted=true; ref_src=; auth_id=xxxxxxxxx; fp=c1ecc123a1924d25bfcd9ff29e27a564; sess=8gesu09bldbosqmf7rnsc2kfmq; st=c3cec1c214ce331114253bd9979b9185c32752afa2d4a6dc30b4f9b8da1f4d69",
    "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36 Edg/92.0.902.67",
    "email": "",
    "password": "",
    "hashed": false,
    "support_2fa": true,
    "active": true
  }
}
```

**Comment by chenks** [comment]
still an issue, guessing this tool no longer works?

**Comment by LunarPenguin** [comment]
It still works for me but you seem to have a strange problem. maybe you can uninstall and reinstall tqdm?

**Comment by chenks** [comment]
tried that 

```
C:\of>pip uninstall tqdm
Found existing installation: tqdm 4.62.0
Uninstalling tqdm-4.62.0:
  Would remove:
    c:\users\chenk\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\scripts\tqdm.exe
    c:\users\chenk\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\site-packages\tqdm-4.62.0.dist-info\*
    c:\users\chenk\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\site-packages\tqdm\*
Proceed (Y/n)? y
  Successfully uninstalled tqdm-4.62.0

C:\of>pip install tqdm
Collecting tqdm
  Using cached tqdm-4.62.0-py2.py3-none-any.whl (76 kB)
Requirement already satisfied: colorama in c:\users\chenk\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\site-packages (from tqdm) (0.4.4)
Installing collected packages: tqdm
  WARNING: The script tqdm.exe is installed in 'C:\Users\chenk\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed tqdm-4.62.0

C:\of>
```

made no difference, same error

**Comment by bigbluedeer** [comment]
possible duplicate of #1455

**Comment by chenks** [comment]
> possible duplicate of #1455

maybe, but hasn't been resolved in either this or the other post.

**Comment by chenks** [comment]
it appears that it's the verison of tqdm causing the issue.
fix is it manually downgrade tqdm back to 4.59.0
